vendor.conf

```diff
@@ -40,6 +39,7 @@ github.com/sirupsen/logrus v1.0.3
 github.com/spf13/cobra 7b2c5ac9fc04fc5efafb60700713d4fa609b777b
 github.com/spf13/pflag 97afa5e7ca8a08a383cb259e06636b5e2cc7897f
 github.com/stretchr/testify 4d4bfba8f1d1027c4fdbe371823030df51419987
+github.com/theupdateframework/notary 8a1de3cfc3f1408e54d6364fc949214a4883a9f3
```

Comment:
this needs to be updated to `05985dc5d1c71ee6c387e9cd276a00b9d424af53` (the commit that moves the import paths) or later cc @endophage

Reply:
Thanks! Forgot about references inside notary itself 🤦‍♂️